### ISSUE TITLE
Allow %c client ID substitution in dynamic security plugin ACLs to match the functionality of ACLs in the standard configuration

### ISSUE THREAD

**Issue by Sammmmm** [eclipse/mosquitto#3191]
Not having this is a large burden for those of us wanting to limit the impact and reach of a compromised client, as we need to create ACLs per client.

**Comment by ralight** [comment]
Agreed, this is an important addition. It's already in the develop branch that will be part of the 2.1 release.

**Comment by ralight** [comment]
Closing due to the 2.1 release
